index.html

```diff
@@ -24,24 +43,24 @@ <h2>The Past</h2>
                 <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
             </div>
         </section>
-        
+
         <section class="middle-content">
-        
+
             <h2>Why Did It Have To Be Boxes...</h2>
-            
+
             <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
+                <div class="box teal">Box 1</div>
+                <div class="box gold">Box 2</div>
+                <div class="box cadetblue">Box 3</div>
```

Comment:
Noble attempt, but a color as a class name will sadly not work. In this case, for each of the boxes you want all the boxes the same size, but you want them different colors. One of the ways you could achieve this is by keeping the "box" class that assigns that specific element its shape and add an additional class to each box to specify the color of the box. For example:

```
<div class="boxes">
    <div class = "box box1">Box 1</div>
    <div class = "box box2">Box 2</div>
    <div class = "box box3">Box 3</div>
</div>
```
This allows you to reuse your CSS styles for the "box" class, while "box X" is the specified color.
Nonetheless, great job!